src/Quadrature.h

```diff
@@ -178,12 +182,20 @@ class LeonardPolarQuad: public Quadrature {
 class GLPolarQuad: public Quadrature {
 
 private:
+  /** the roots to the Legendre polynomial of degree _num_polar */
+  std::vector <double> _roots;
```

Comment:
If we can prove that correcting the weights is advantageous there's no reason to keep the other option around. But let's put off merging that until we convince Ben that is the case.